core/src/main/python/wlsdeploy/tool/deploy/applications_deployer.py

```diff
@@ -643,6 +643,11 @@ def __build_app_deploy_strategy(self, location, model_apps, existing_app_refs, s
                                                                                     from_archive=True)
 
                 existing_app_ref = dictionary_utils.get_dictionary_element(existing_app_refs, versioned_name)
+                if len(dictionary_utils.get_element(app_dict, TARGET)) == 0:
```

Comment:
What does get_element return if TARGET is not present?  Does not present get the same result as zero length string?

Reply:
This PR has been closed and no longer needed